sui_mamoru_relay/Move.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "sui_mamoru_relay"
+version = "0.0.1"
+
+[dependencies]
+Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "devnet" }
+
+[addresses]
+sui_mamoru_relay = "0x0"
```

Comment:
what addreses of incident registry and validators module?

Reply:
The module is not published yet. Should I publish it already?